### PR TITLE
Flex related bugfix + dev goerli config and flex ABI updated

### PR DESCRIPTION
**Pull request by lyaunzbe**
minor bugfix related to Flex contract after local testing:
- casting externalDependencyAssetCount to BigInt

also, after merging https://github.com/ArtBlocks/artblocks-contracts/pull/303 we have updated Flex contract ABI + re-generated codegen + updated dev goerli with new deployment addresses

### CODE REVIEW

File: src/mapping-v2-core.ts [hunk 1]
```diff
@@ -288,12 +295,13 @@ export function handleExternalAssetDependencyRemoved(
...
   if (!contractEntity) {
     log.warning("Contract not found for GatewayUpdated event", []);
     return;
   }
-  const dependencyType = FLEX_CONTRACT_EXTERNAL_ASSET_DEP_TYPES[event.params._dependencyType];
+  const dependencyType =
+    FLEX_CONTRACT_EXTERNAL_ASSET_DEP_TYPES[event.params._dependencyType];
   if (dependencyType === "IPFS") {
     contractEntity.preferredIPFSGateway = event.params._gatewayAddress;
   } else {
```
Comment (ryley-o) [inline src/mapping-v2-core.ts:307]: Do we need to clear the dependency not being set (e.g. clear arweave if setting IPFS)?

**Review by ryley-o — APPROVED** [review]
🙌 🙌  LGTM

File: src/mapping-v2-core.ts (hunk 1 above)
Comment (lyaunzbe) [reply src/mapping-v2-core.ts:307]: a project can have assets from both IPFS and Arweave, so I don't think we'd want to clear

Comment (ryley-o) [reply src/mapping-v2-core.ts]: ahhhh makes sense! SGTM!

**Review by yoshiwarab — APPROVED** [review]

**Pull request merged**